sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+# Release History
+## 1.0.0 (Unreleased)
```

Comment:
1.0.0-beta-001?

Reply:
if I remember this correctly, they have already gone GA right?

Reply:
Service is GAed, yes. Given this is a changelog, it will have to start as 1.0.0-beta-1 as David mentioned.

Reply:
Even if they GA'd they didn't with this package name.